MyReadingList/MyReadingList/MyReadingListView.swift

```diff
@@ -10,21 +10,16 @@ import SwiftUI
 
 struct MyReadingListView: View {
     var body: some View {
-        VStack {
-            Text(NSLocalizedString("ui.myReadingList.title", value: "My Reading List", comment: "title for the MyReadingList screen"))
-                .font(.headline)
-            Spacer()
-            
+        NavigationView {
             List {
-                HStack{
-                    Button {
-                        //do nothing for now
+                HStack {
+                    NavigationLink {
+                        SearchView()
                     } label: {
-                        Image(systemName: "plus.circle")
+                        Text(NSLocalizedString("ui.myReadingList.addABook.button", value: "Add a book", comment: "Add a Book button in MyReadingList screen"))
```

Comment:
Personally, I think the code will be easier to read if you put each parameter in the localized string on a newline. Such as :
```
Text(NSLocalizedString("ui.myReadingList.addABook.button", 
value: "Add a book", 
comment: "Add a Book button in MyReadingList screen")
```
(let the editor take care of the proper indentation)

But that is entirely up to you. If you think it's easy to read as is, then leave it as is.